gdx-runtime/build.gradle

```diff
@@ -1,14 +1,16 @@
 apply plugin: "java"
+apply plugin: "maven-publish"
+
+group = 'com.github.jpooleycodes.mundus'
+version = '0.3.0'
 
 sourceSets.main.java.srcDirs = ["src/"]
 sourceSets.main.resources.srcDirs = ["src/"]
 
-def VERSION = '0.3.0'
 def NAME = 'mundus-gdx-runtime'
 
 task distRuntime(type: Jar, dependsOn: classes) {
-    baseName = NAME
-    version = VERSION
+    archivesBaseName = NAME
 
     dependsOn configurations.runtimeClasspath
     from {
```

Comment:
Should we hold this gradle task if there is `publishToMavenLocal` task via maven-publish plugin.

Reply:
We can get rid of but will need to update the working from sources wiki page which refers to them.